profiles/common/modules/features/nexteuropa_dgt_connector/tmgmt_poetry/plugins/tmgmt_poetry.plugin.inc

```diff
@@ -80,8 +80,12 @@ class TMGMTPoetryTranslatorPluginController extends TMGMTDefaultTranslatorPlugin
    * and the number of items.
    */
   public function canTranslate(TMGMTTranslator $translator, TMGMTJob $job) {
-    // Check that the item does not have pending requests.
+    // Check that only one item is being requested.
     $items = $job->getItems();
+    if (count($items) > 1) {
+      return FALSE;
+    }
+    // Check that the item does not have pending requests.
```

Comment:
If lines 85 to87 are the only difference, I suggest to remove the whole code below and replace by `return parent::canTranslate($translator, $job)`. That can ease the maintenance.

Reply:
For that we would need that this class was extending TmgmtDgtConnectorTranslatorPluginController (see my first comment in this review).

I also like your suggestion, but because we want to avoid many changes in "tmgmt_poetry", I will not do it for now. May be we should ask for a third opinion?

PS: This changes in "tmgmt_poetry" are meant to revert the changes made in [NEPT-873](https://webgate.ec.europa.eu/CITnet/jira/browse/NEPT-873?focusedCommentId=2267558&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-2267558) (and leave it as it was before).